### PR TITLE
docs: bulk spelling edits

### PR DESCRIPTION
**Pull request by ElPaisano**
**This PR doesn't change any code, only plain text and markdown file text, so wasn't clear if I needed to follow https://github.com/ipfs/community/blob/master/CONTRIBUTING_GO.md#go-contribution-guidelines       Sorry...**

Ran a spell check script against txt and md files in the repo. Wasn't sure about a few words (they might be special terms or code stuff, etc.), so please let me know if I need to change anything back.

### CODE REVIEW

**Comment by ElPaisano** [comment]
https://circleci.com/gh/ipfs/kubo/94436 failed for some reason... not sure if this is expected

**Review by lidel — APPROVED** [review]
Thank you @ElPaisano!

**Comment by ElPaisano** [comment]
@lidel a few tests still failed on rerun

File: assets/init-doc/security-notes [hunk 1]
```diff
@@ -12,7 +12,7 @@ Please note the following:
 - ipfs is a networked program, and may have serious undiscovered
   vulnerabilities. It is written in Go, and we do not execute any
   user provided data. But please point any problems out to us in a
-  github issue, or email security@ipfs.io privately.
+GitHub issue, or email security@ipfs.io privately.
```
Comment (Jorropo) [inline assets/init-doc/security-notes:15]: This is a `<li>`
```suggestion
  GitHub issue, or email security@ipfs.io privately.
```

**Review by lidel — APPROVED** [review]

File: assets/init-doc/security-notes [hunk 2]
```diff
@@ -12,7 +12,7 @@ Please note the following:
 - ipfs is a networked program, and may have serious undiscovered
   vulnerabilities. It is written in Go, and we do not execute any
   user provided data. But please point any problems out to us in a
-  github issue, or email security@ipfs.io privately.
+  GitHub issue, or email security@ipfs.io privately.
```
Comment (lidel) [inline assets/init-doc/security-notes:15]: We don't want to change hashes of these files.
```suggestion
  github issue, or email security@ipfs.io privately.
```

**Pull request merged**